src/components/00-materials/README.md

```diff
@@ -88,13 +88,30 @@ npm install @svgr/webpack --save-dev
 
 ## Contribution
 
-### Process of adding a new icon/image
+### Process of adding a new image
 
 1. Involve a designer by sending him the new SVG in question, unless the designer already has the file. He will review it and possibly change some things, for example by adjusting inner padding etc. in order to guarantee overall quality.
-2. Remove unnecessary code inside the SVG:
+1. Remove unnecessary code inside the SVG:
    - a. `data-name="Layer 2"` # layer info for vector drawing software
    - b. `<path fill="#fff" d="M0 0h96v96H0z"/>` # outer bounding box filled with white background
-3. - a. use the internal [svgo](https://github.com/svg/svgo) -invoking scripts `npm run build-icons` or `npm run build-images` from materials itself to clean up and optimize the SVGs, then copy the optimized files back from the `.tmp` folder to `icons-raw` or `images-raw`
+1. - a. use the internal [svgo](https://github.com/svg/svgo) -invoking script `npm run build-images` from the materials-folder itself to clean up and optimize the SVGs, then copy the optimized files back from the `.tmp` folder to `images-raw`
```

Comment:
What about the icons? They handled by your new script?

Reply:
yes, mentioned in the next paragraph.